### PR TITLE
change_numa_tuning: add case for changing numatune

### PR DESCRIPTION
**Pull request by dzhengfy**
Add case to test changing numatune

Case ID: xxx-297113

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
```
Test with libvirt 9.3.0
 (1/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_strict: WARN: Test passed but there were warnings during execution. Check the log for details. (84.63 s)
 (2/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_interleave: WARN: Test passed but there were warnings during execution. Check the log for details. (118.88 s)
 (3/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_preferred: WARN: Test passed but there were warnings during execution. Check the log for details. (87.89 s)
 (4/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_restrictive: WARN: Test passed but there were warnings during execution. Check the log for details. (95.17 s)

```

**Comment by dzhengfy** [comment]
Test on libvirt-8.0.0
```
 (1/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_strict: PASS (33.20 s)
 (2/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_interleave: PASS (33.64 s)
 (3/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_preferred: PASS (34.69 s)
 (4/4) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.change_numa_tuning.mem_mode_restrictive: PASS (38.36 s)
```

File: provider/numa/numa_base.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 #   Author: Dan Zheng <dzheng@redhat.com>
 #
+import re
```
Comment (cliping) [inline provider/numa/numa_base.py:10]: Import re and process twice.

File: provider/numa/numa_base.py [hunk 2]
```diff
@@ -142,6 +146,16 @@ def get_nodeset_from_numad_advisory(self):
         return numad_ret
 
 
+def get_host_numa_memory_alloc_info(mem_size, test):
```
Comment (cliping) [inline provider/numa/numa_base.py:149]: Please add docstring.

**Comment by cliping** [comment]
Others LGTM

File: provider/numa/numa_base.py (hunk 1 above)
Comment (dzhengfy) [reply provider/numa/numa_base.py:10]: Updated

File: provider/numa/numa_base.py (hunk 2 above)
Comment (dzhengfy) [reply provider/numa/numa_base.py:145]: updated

**Review by cliping — APPROVED** [review]
LGTM

**Review by chloerh — COMMENTED** [review]
The rest is ok to me

**Review by cliping — APPROVED** [review]

**Review by chloerh — APPROVED** [review]

**Pull request merged**